### ISSUE TITLE
Display ghost_text only when completion item is selected

### ISSUE THREAD

**Issue by ayles** [hrsh7th/nvim-cmp#1916]
Currently ghost text is displayed whenever cmp pop-up is present and when no completion item is selected first one is used. I believe it is somewhat consistent with `confirm { select = true }`, but for users with `select = false` it may be counterintuitive.

**Comment by pidgeon777** [comment]
Yes, please add an option to show ghost_text only when an item is selected.
